configs/camp-co-reality.js

```diff
@@ -0,0 +1,137 @@
+module.exports = {
+  template: "themecamp",
+  host: {
+    url: "https://co-reality.co",
+    name: "Co-reality Collective",
+    icon: "/collective-icon.png",
+  },
+  description: {
+    name: "Welcome to Camp Co-reality.",
+    // program_url: "TODO",
+  },
+  name: "Camp Co-reality",
+  map_url:
+    "https://firebasestorage.googleapis.com/v0/b/co-reality-staging.appspot.com/o/users%2FYFLe3gR6LnPwn6sCWqgys1aXbDG2%2Fimages%2FSPA_050820_Camp_Co_Reality_Blank_120820_1345.png?alt=media&token=4d55eb5b-e0ee-4477-871b-8c09ceb013c3",
+  owners: ["tpl7htc3ceOgWI5J4xdLTNOfehh1"], // staging
+  rooms: [
```

Comment:
Do we keep rooms or should these become Venues (I think structure should be similar for PartyMap and Camp)